### DISCUSSION TITLE
How to perform recognition (only), knowing the location of text(similar location everytime)

### DISCUSSION THREAD

**Discussion by WilfriedPonnou** [mindee/doctr#1142] [Q&A]
Hi everyone, as specified in the title, i wanted to know how i can make recognition inference only (with my custom trained model:crnn_mobilenet_v3_small base) knowing that my text is always located in the same area of the images that i have (i have the coordinates of the text location).
Thanks in advance

**Comment by AbrahamIES** [comment]
1 . you first need to create instance of an recognition model 
2.- load your state dict of your trained model
3.- crop the image with the particular coordinates you have
4.- run the reco model with your cropped image
